pontoon/contributors/templates/contributors/settings.html

```diff
@@ -65,6 +65,11 @@ <h3>Personal information</h3>
             {{ user_profile_form.username.errors }}
             <p class="help">{{ user_profile_form.username.help_text }}</p>
         </div>
+        <div class="field">
+            {{ user_profile_form.bio.label_tag(label_suffix='') }}
+            {{ user_profile_form.bio }}
+            {{ user_profile_form.bio.errors }}
+        </div>
```

Comment:
This seems rather terse, if/as we're displaying only the string "Bio" as a descriptor here. We should note here at least that this information will (eventually) be displayed publicly on the profile page.